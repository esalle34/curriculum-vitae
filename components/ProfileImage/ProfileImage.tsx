const ProfileImage = () => (
  <img
    className="md:w-44 self-center mb-8 rounded shadow-xl"
    src="/images/eric-salle.jpg"
  />
);

export default ProfileImage;
