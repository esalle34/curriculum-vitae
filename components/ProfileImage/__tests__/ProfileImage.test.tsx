import { render } from "@testing-library/react";
import ProfileImage from "../ProfileImage";

describe("ProfileImage", () => {
  it("should match snapshot correctly", () => {
    const { container } = render(<ProfileImage />);
    expect(container).toMatchSnapshot();
  });
});
