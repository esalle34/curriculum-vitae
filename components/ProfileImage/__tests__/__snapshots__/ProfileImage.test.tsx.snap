// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileImage should match snapshot correctly 1`] = `
<div>
  <img
    class="md:w-44 self-center mb-8 rounded shadow-xl"
    src="/images/eric-salle.jpg"
  />
</div>
`;
